packages/css/src/components/field/field.scss

```diff
@@ -8,7 +8,18 @@
   break-inside: avoid;
   display: flex;
   flex-direction: column;
-  gap: var(--ams-field-gap);
```

Comment:
This removal stops a possible custom value for the `gap` token from working. This PR deprecates the token, but we promised that deprecated features will continue to work for two more major releases.

Should we restore this line of CSS (and add a comment) and set our own value of the token to `initial`?

If so, we should also find a way to remember that we could or should change `display: flex` back to `block`, as we no longer use or need Flexbox and `block` is the default.

If not, we can revert to `display: block` now.